rotkehlchen/api/rest.py

```diff
@@ -3057,6 +3057,15 @@ def get_liquity_staked(self, async_query: bool) -> Response:
             addresses=self.rotkehlchen.chain_manager.queried_addresses_for_module('liquity'),
         )
 
+    def get_liquity_stability_pool(self, async_query: bool) -> Response:
```

Comment:
Shouldn't the name be the same? Otherwise it's confusing for the reader

```suggestion
    def get_liquity_stability_pool_positions(self, async_query: bool) -> Response:
```